packages/synapse-react-client/src/components/AccountLevelBadges/AccountLevelBadges.test.tsx

```diff
@@ -39,105 +60,141 @@ const getMockUserBundle = (
 const queryForBadgeLabel = (badgeType: AccountLevelBadgeType) => {
   return screen.queryByText(accountLevelBadgeConfig[badgeType].label)
 }
+const props = {
+  userId: USER_ID,
+}
 
-describe('AccountLevelBadges', () => {
-  const props = {
-    userId: '1234',
-  }
+function renderComponent() {
+  const component = render(<AccountLevelBadges {...props} />, {
+    wrapper: createWrapper(),
+  })
 
-  function renderComponent() {
-    return render(<AccountLevelBadges {...props} />, {
-      wrapper: createWrapper(),
-    })
-  }
+  const user = userEvent.setup()
+
+  return { component, user }
+}
 
+describe('AccountLevelBadges', () => {
   beforeEach(() => {
     jest.clearAllMocks()
+    mockGetMyUserBundle.mockResolvedValue(mockUserBundle)
   })
 
   it('registered user', () => {
-    mockGetUserBundle.mockResolvedValueOnce(getMockUserBundle(false, false))
+    mockGetUserBundle.mockResolvedValue(getMockUserBundle(false, false))
 
     renderComponent()
 
-    expect(queryForBadgeLabel('certified')).toBeNull()
-    expect(queryForBadgeLabel('validated')).toBeNull()
-    expect(queryForBadgeLabel('enabledMFA')).toBeNull()
+    expect(queryForBadgeLabel('certified')).not.toBeInTheDocument()
```

Comment:
Use `toBeInTheDocument`, which tests the same thing, but better explains what this code is doing